specification.markdown

```diff
@@ -223,6 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |:---------------|:---------|:-----------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| parent         | Yes      | The path to the parent EBML Element. This path MUST be defined with the full hierarchy of EBML Elements separated with a `/`. The top EBML Element in the path hierarchy being the first in the value. Elements that can be found anywhere inside en EBML Element and its children MUST use the `*` (wildcard) value after the path of the first parent allowed. |
```

Comment:
Re:
> Elements that can be found anywhere inside en EBML Element and its children MUST use the `*` (wildcard) value after the path of the first parent allowed.

I would prefer to express multiple Parent Elements in this case. Such as, for SimpleTag, `parent="Tag|SimpleTag"`.

Although having multiple values concatenated within an attribute is bad XML form. Perhaps better to use like:

```xml
<element name="SimpleTag" cppname="TagSimple" level="3" id="0x67C8" type="master" minOccurs="1" maxOccurs="unbounded" minver="1" webm="0">
   <parent>Tag</parent>
   <parent>SimpleTag</parent>
   <documentation lang="en">Contains general information about the target.</documentation>
</element>
```

Removed the `recursive` attribute in the example above since the `parent` sub-element implies it.